### PR TITLE
android-sms-backup-and-restore: Add version 2.14.37

### PR DESCRIPTION
**Pull request by digrouz**

### CODE REVIEW

**Comment by digrouz** [comment]
License info are here: https://www.backuptrans.com/terms-of-use.html I was not sure what to put for it.

**Comment by digrouz** [comment]
you're right, i've removed shortcut for device manager

**Comment by digrouz** [comment]
changes applied

**Comment by Ash258** [comment]
> BY USING THE Backuptrans **WEB SITE**, YOU AGREE TO THESE TERMS OF USE. IF YOU DO NOT AGREE TO THESE TERMS OF USE, PLEASE DO NOT USE THE Backuptrans **WEB SITE**.

backuptrans.com/terms-of-use.html 

I would rather go with `Unknown` license. Appliation do not have any license inside directory, on web I cannot find any license agreement aimed for this specific application. And terms-of-use looks like it is only for webpage.

File: bucket/android-sms-backup-and-restore.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.14.37",
+    "description": "Backup & Restore Android SMS on computer with ease.",
+    "homepage": "https://www.backuptrans.com/android-sms-backup-restore.html",
+    "license": "Unknow",
```
Comment (Ash258) [inline bucket/android-sms-backup-and-restore.json]: https://dictionary.cambridge.org/dictionary/english/unknown

```suggestion
    "license": "Unknown",
```

**Pull request merged**